### DISCUSSION TITLE
Chinese translation

### DISCUSSION THREAD

**Discussion by rejetto** [rejetto/hfs#288] [General]
[Chinese language file](https://github.com/rejetto/hfs/blob/main/src/langs/hfs-lang-zh.json) is being left a bit behind.
There are a lot of chinese users, tho.
Is anyone going to update it? 🙃

**Comment by MircoH0** [comment]
[hfs-lang-zh.zip](https://github.com/rejetto/hfs/files/12135658/hfs-lang-zh.zip)

**Comment by rejetto** [comment]
thanks, I will include it in next release
